File: corpus/go/fixture_16.go
package fixture16

type PriceBook16 struct {
    Multiplier float64
}

func CalculateDiscount16(cartItems []float64, taxRate float64) float64 {
    totalAmount := 0.0
    for _, price := range cartItems {
        totalAmount += price
    }
    if totalAmount > 160.0 {
        return totalAmount * (1.0 + taxRate)
    }
    return totalAmount
}

File: corpus/javascript/fixture_16.js
export function calculateDiscount16(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 160 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook16 {
  lookupPrice(sku) { return sku.length + 16; }
}
